go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/go-logfmt/logfmt v0.4.0 // indirect
 	github.com/go-redis/redis/v7 v7.0.0-beta.5
 	github.com/go-stack/stack v1.8.0 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
```

Comment:
I _think_ the changes in `go.mod` and `go.sum` are accidental and could be reverted.

Reply:
Good catch, indeed.